File: Programação_Funcional/Listas/Estrutura_Condicional/Ex1036.clj
(defn bhaskara [a b c]
  (let [delta (- (Math/pow b 2) (* 4 a c))]
    (if (or (= a 0.0) (<= delta 0))
      (println "Impossivel calcular")
      (let [r1 (/ (+ (- b) (Math/sqrt delta)) (* 2 a))
            r2 (/ (- (- b) (Math/sqrt delta)) (* 2 a))]
        (println (str "R1 = " (format "%.5f" r1)))
        (println (str "R2 = " (format "%.5f" r2)))))))

(bhaskara (read) (read) (read))

File: Estrutura_Condicional/Python/Ex1036.py
import math

def calcular_raizes(a, b, c):
    delta = b**2 - 4*a*c
    
    if delta < 0 or a == 0:
        print("Impossivel calcular")
        return
    
    r1 = (-b + math.sqrt(delta)) / (2*a)
    r2 = (-b - math.sqrt(delta)) / (2*a)
    
    print(f"R1 = {r1:.5f}")
    print(f"R2 = {r2:.5f}")

a, b, c = map(float, input().split())

calcular_raizes(a, b, c)